### DISCUSSION TITLE
Install Modules

### DISCUSSION THREAD

**Discussion by gmesika** [cosmos/community#25] [Announcements]
I'm running cosmos-sdk with wasm built-in
https://github.com/CosmWasm/wasmd/tree/v0.32.1
and I want to install on top the:
https://pkg.go.dev/github.com/evmos/ethermint@v0.19.3
I made sure it is pretty close in dependencies - otherwise it is incredible mess

go get github.com/evmos/ethermint@v0.19.3
then it asked: 
go get github.com/cosmos/cosmos-sdk/server/grpc@v0.45.15
once I did that , it errored:
go: module github.com/golang/protobuf is deprecated: Use the "google.golang.org/protobuf" module instead.

so I have added to go.mod in the replace section:
        github.com/golang/protobuf => google.golang.org/protobuf v1.31.0

eventually this is the change in go.mod
 (+)       github.com/cenkalti/backoff/v4 v4.1.1 // indirect
 (+)       github.com/evmos/ethermint v0.19.3 // indirect
 (-)       github.com/improbable-eng/grpc-web v0.14.1 // indirect 
 (+)       github.com/improbable-eng/grpc-web v0.15.0 // indirect
 (+)       github.com/onsi/gomega v1.19.0 // indirect

managed to build the project !
but the x/ directory didnt add the evm , and network didn't get any change to support the evm

any idea ? should I manually update the app.go?

**Comment by julienrbrt** [comment]
You can take a look in the ethermint repo how they wire their app.go:
https://github.com/evmos/ethermint/blob/v0.19.3/app/app.go (for your version)
